src-cpp/rdkafkacpp.h

```diff
@@ -548,6 +549,15 @@ class Consumer : public virtual Handle {
                                 int timeout_ms,
                                 ConsumeCb *consume_cb,
                                 void *opaque) = 0;
+
```

Comment:
Can you make this a private method outside rdkafkacpp.h so that it only contains virtual methods?